scrapy/extensions/httpcache.py

```diff
@@ -220,6 +224,8 @@ def open_spider(self, spider):
         dbpath = os.path.join(self.cachedir, '%s.db' % spider.name)
         self.db = self.dbmodule.open(dbpath, 'c')
 
+        logger.debug("Using DBM cache storage in %(cachepath)s" % {'cachepath': dbpath}, extra={'spider': spider})
```

Comment:
That's not really the case, see `scrapy.spiders.Spider.__str__` (in the __init__.py)